content/config/containers/resource_constraints.md

```diff
@@ -234,7 +234,9 @@ for real-time tasks per runtime period. For instance, with the default period of
 containers using the real-time scheduler can run for 950000 microseconds for every
 1000000-microsecond period, leaving at least 50000 microseconds available for
 non-real-time tasks. To make this configuration permanent on systems which use
-`systemd`, see [Control and configure Docker with systemd](../daemon/systemd.md).
+`systemd`, create a systemd unit file for the `docker` service. For an example,
```

Comment:
probably the correct term is either "drop-in" or "override" file for this (I've seen "drop-in" being commonly used as term, but not sure if both are technically the same).

Reply:
I think unit file is more correct; https://access.redhat.com/documentation/enus/red_hat_enterprise_linux/8/html-single/using_systemd_unit_files_to_customize_and_optimize_your_system/index

Reply:
I think 'drop-in' is indeed the correct term here (and in `content/config/daemon/proxy.md`). 'Unit file' is a generic term, while 'drop-in' specifically designates a file which is overriding a base 'unit file'. See https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#.

Reply:
thanks for the reference! I've reverted this change